crates/nu-protocol/src/parse_error.rs

```diff
@@ -413,6 +413,10 @@ pub enum ParseError {
     #[error("{0}")]
     #[diagnostic()]
     LabeledError(String, String, #[label("{1}")] Span),
+
+    #[error("{0}")]
+    #[diagnostic(help("{1}"))]
+    LabeledErrorWithHelp(String, String, String, #[label("{2}")] Span),
```

Comment:
Can we make this a named struct variant instead of a tuple struct variant, else it is hard to follow what goes where.
```
    LabeledErrorWithHelp{
    	msg: String, 
    	help: String, 
    	label: String, 
    	#[label("{label}")] 
    	span: Span
    	},
```

Reply:
fixed

Reply:
Awesome thanks for your quick turnaround!